_includes/alert.html

```diff
@@ -4,18 +4,17 @@
     usa-alert--{{ include.variant }}
     {% if include.slim %}usa-alert--slim{% endif %}
     {% if include.icon == false %}usa-alert--no-icon{% endif %}
-    {% if include.classNames %}{{ include.classNames }}{% endif %}
-  "
+    {% if include.classNames %}{{ include.classNames }}{% endif %}"
   {% if include.role %}role="{{ include.role }}"{% endif %}
 >
   <div class="usa-alert__body">
     {% if include.heading %}
-      <h4 class="usa-alert__heading">{{ include.heading }}</h4>
+      {{ include.heading }}
     {% endif %}
     {% if include.text %}
-      <p class="usa-alert__text">
+      <div class="usa-alert__text">
         {{ include.text }}
-      </p>
+      </div>
```

Comment:
These changes let us put whatever formatting we want into the specific component sections here. This lets us:

Specify which heading level to use in context, so we're not fighting with using the wrong heading level when using accordions
Use more complex accordion content. If, for example, we wanted to use two paragraphs, instead of raw text inside the one paragraph before, this update allows us to do that